caserver/caserver.go

```diff
@@ -357,6 +326,73 @@ func dsGet(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Key update request handler
+func kurPost(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
Can we do the same thing as the other request, and accept/reply with cbor as well. It's going to be quite a bit more efficient to process on the target that way.

Reply:
The ccs endpoint does handle both. ‘Kur’ here isn’t implemented yet, but was moved in this PR so that functions are implemented in the same order as the header file.

Reply:
See the readme for an example of both json and CBOR being used for ccs.